src/timeline-report.js

```diff
@@ -291,7 +298,7 @@ export class TimelineReport extends StacheElement {
       this.filteredDerivedIssues,
       this.routeData.normalizeOptions,
       this.rollupTimingLevelsAndCalculations,
-      new Date(new Date().getTime() - this.compareToTime.timePrior)
```

Comment:
this.compareToTime.timePrior doesn't exist on this class. I assume there's some canJS magic happening here. `compareToTime` comes from the old canjs slider. When this element is removed from the dom, the application crashes.

To prevent that I found the source for `compareToTime.timePrior` and added it here by referencing `this.routeData.compareTo`

I'm not sure this is correct, but it's functioning now

Reply:
cc: @justinbmeyer